DeviceDiagnostics/DeviceDiagnostics.cpp

```diff
@@ -22,13 +22,36 @@
 #include <curl/curl.h>
 #include <time.h>
 
-#include "utils.h"
-
 #define DEVICE_DIAGNOSTICS_METHOD_NAME_GET_CONFIGURATION  "getConfiguration"
 #define DEVICE_DIAGNOSTICS_METHOD_GET_AV_DECODER_STATUS "getAVDecoderStatus"
+#define TEST_REQUEST "{\"paramList\":[{\"name\":\"test\"}]}"
```

Comment:
Pls override the url within the tests instead of doing it here in the plugin source. We can move "127.0.0.1:10999" into a member variable that can be mocked to a different url for testing. Can we also host the mock server locally instead of pointing to a specific endpoint.